### DISCUSSION TITLE
Modeling ERT signals in anisotropic mediums

### DISCUSSION THREAD

**Discussion by alexander-a-vlasov** [gimli-org/pyGIMLi#522] [Q&A] [answered]
Hello,
I know that the gimli engine supports aniotropic mediums. How to set anisotropic volume (different horizontal and vertical resistivity) or apply anisotropic  matrix for ERT using Pygimli? I am using a 2D and 3D simulator.

**Comment by carsten-forty2** [comment, marked as answer]
Currently, there is only support for anisotropic media for the generic finite element simulator but not for the ERT modules.

**Reply by alexander-a-vlasov** [reply]
Thank you for your answer. I will hope that in the future there will be anisotropic mediums for ERT simulation and look forward to reading the release notes for new version pygimly

**Comment by halbmy** [comment]
You can model anisotropic resistivity with the finite element solver described on
https://www.pygimli.org/pygimliapi/_generated/pygimli.solver.html#pygimli.solver.solveFiniteElements
In 3D, you just need some loops and bookkeeping to collect the potentials. In 2D you have to follow the wavenumber scheme in
https://www.pygimli.org/_examples_auto/3_dc_and_ip/plot_04_ert_2_5d_potential.html

I guess it could even make sense to open the `ert.simulate` API for anisotropy. But I don't see it for inversion.
